packages/nextjs/app/ipfsDownload/page.tsx

```diff
@@ -18,20 +18,28 @@ const IpfsDownload: NextPage = () => {
 
   const handleIpfsDownload = async () => {
     setLoading(true);
-    const notificationId = notification.loading("Getting data from IPFS");
-    try {
-      const metaData = await getMetadataFromIPFS(ipfsPath);
-      notification.remove(notificationId);
-      notification.success("Downloaded from IPFS");
-
-      setYourJSON(metaData);
-    } catch (error) {
-      notification.remove(notificationId);
-      notification.error("Error downloading from IPFS");
-      console.log(error);
-    } finally {
-      setLoading(false);
+    const notificationId = notification.loading("Getting data from IPFS...");
+    let attempt = 0;
+    const maxAttempts = 2;
```

Comment:
Would prefer to extract this in a separate `constants` file with `SCREAMING_SNAKE_CASE_NOTATION`

Reply:
Same case as the upload page too